examples/storage/couchdb/README.md

```diff
@@ -0,0 +1,84 @@
+allow for clustering within Kubernetes.  There are currently three config
+files `vm.args`, `local.ini` and `default.ini` that are stored as blobs in a
+[ConfigMap](http://kubernetes.io/docs/user-guide/configmap/) found at 
+`core-config-maps/core-configmap.yaml`. The `vm.args` file is where we set the
+erlang 'magic' cookie for all the couchdb instances alowing them to 
+communicate with each other.  Before we get to creating our couchdb cluster 
+these config maps need to be created:
+```
+kubectl create -f core-config-maps/core-configmap.yaml -f core-config-maps/cluster-adm.yaml 
+configmap "core-config" created
+secret "cluster-adm" created
+```
+#### Note on Admin users
+We have populated the `local.ini` config file with user "admin" and the plain 
+text password "changeme". On startup couchdb will hash the users password.  If 
+you would like to define your own admin users then just add them to local.ini 
+section of the core-config.  Also make sure that there is a valid admin user 
+credentials in the cluster admin secret at `core-config-maps/cluster-adm.yam`.
+The username and password values need to be base64 encoded checkout the [Secret](http://kubernetes.io/docs/user-guide/secrets/)
+docs for more info. This is needed for the utility pods if misconfigured we
+are going to have a bad time. 
+
+### Creating the services 
+
+Along with the erlang cookie, Couchdb instances need to have a static network 
+identity.  This static identity is used to set the erlang node names for the 
+couchdb instances.  To achieve this in Kubernetes petsets are leveraged giving 
+every couchdb pod its own DNS name for its lifetime. 
+
+There are two services we need to create for our couchdb cluster.  First 
+theres the headless service for pod to pod discovery using DNS lookup and to 
+give static hostname. Then we have the service that exposes the couchdb cluster. Let us create them:
+
+```
+kubectl create -f couchdb-petset-headless-service.yaml -f couchdb-petset-service.yaml 
+service "kubetest001-backend" created
+service "kubetest001" created
+```
+
+### Creating the PetSet
+Now that we have the necessary configmaps, secrets and the services created 
+let's create the couchdb petset!
+```
+kubectl create -f couchdb-petset.yaml 
+petset "kubetest001" created
+```
+You can now head over to the browser and checkout the kubernetes dashboard to
+see the newly create couchdb cluster if you have it running in your kubernetes cluster.
+Alternitively we can see the couchdb pods:
+```
+kubectl get pods --namespace cloudant
+NAME                                     READY     STATUS    RESTARTS   AGE
+kubetest001-0                            3/3       Running   0          3m
+kubetest001-1                            3/3       Running   0          4m
+kubetest001-2                            3/3       Running   0          4m
+```
+
+If your running minikube you can discover the cluster address as 
+follows: 
+```
+minikube service kubetest001 --url
```

Comment:
A `kubectl get service couchdb001` will get the user the external IP of the service. Now it mentioned in the README